test/js/pageobjects/Common.js

```diff
@@ -0,0 +1,16 @@
+import Base from './BasePage.js'
+import {locale} from '../utils/mochaw'
+
+class Common extends Base {
```

Comment:
how is `Common` different from `Base`?

Reply:
The idea for the `Common` class is to have the page objects and functions that are shared across various screens e.g. `verifyTitle` or `backArrow` so we avoid repetitions.

Reply:
but is it different from `Base`? are page objects which might be `Base` but not `Common`?

Reply:
Shall I address it in separate PR? 

@stefaniacardenas already ran the acceptance on that PR so it's ready to be merged. There is also PR for cleaning the Ruby dependencies on top of that branch I won't need to solve conflicts there.

After discussing I understood the changes that would need to be done are:
1. Rename the Common class to Screen
2. Screen extends from Base
3. Other classes in `pageobjects` extend from Screen
4. Move `copy(lang) { return locale(lang) }` to Screen class and remove it from other pageobject classes.

Reply:
I have the stash changes ready to for the above but I don't want to introduce that many changes into the PR that was already reviewed and accepted.